resources/tidb-pdf-generation-tutorial.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Self-service generation of TiDB Documentation PDF tutorial

```

Comment:
```suggestion
title: TiDB Documentation PDF Generation Tutorial
```

Reply:
In the English version, we can skip the word "Self-service" as it sounds unnecessary.